back/engines/commercial/project_folders/spec/services/user_role_service_spec.rb

```diff
@@ -102,4 +102,10 @@
       end
     end
   end
+
+  describe 'moderates_something?' do
+    it 'permits folders moderators' do
+      expect(service.moderates_something?(create(:project_folder_moderator))).to be_truthy
```

Comment:
I consider `be_truthy` to weak. The method returns a boolean, so I would write:

```suggestion
      expect(service.moderates_something?(create(:project_folder_moderator))).to be true
```